src/actions/queries.js

```diff
@@ -12,52 +12,8 @@ export const getCurrentUser = (userId) =>
 
 export const fetchUserCount = () => axios.get("/api/users/count");
 
-export const updateUser = (
-  email,
-  first,
-  last,
-  number,
-  birthday,
-  zip,
-  address,
-  city,
-  state,
-  notes,
-  userId
-) => {
-  var query = "";
-  if (first) {
-    query += "first_name=" + first + "&";
-  }
-  if (last) {
-    query += "last_name=" + last + "&";
-  }
-  if (number) {
-    query += "phone_number=" + number + "&";
-  }
-  if (birthday) {
-    query += "date_of_birth=" + birthday + "&";
-  }
-  if (zip) {
-    query += "zip_code=" + zip + "&";
-  }
-  if (address) {
-    query += "address=" + address + "&";
-  }
-  if (city) {
-    query += "city=" + city + "&";
-  }
-  if (state) {
-    query += "state=" + state + "&";
-  }
-  if (notes) {
-    query += "notes=" + notes + "&";
-  }
-
-  if (query.length > 0) {
-    query = query.slice(0, -1);
-    axios.post(`/api/users/updateUser?email=${email}&${query}`, { userId });
-  }
+export const updateUser = (id, user, role) => {
+  axios.post(`/api/users/${id}`, user, role);;
```

Comment:
Why do you have role as a parameter here?

Reply:
Currently, the role isn't being updated correctly. Can you take a look?

Reply:
Yeah, I am looking into it now

Reply:
This should be resolved